src/app/app.module.ts

```diff
@@ -48,7 +49,13 @@ export function HttpLoaderFactory(http: HttpClient) {
       }
     }),
   ],
-  providers: [],
+  providers: [
+    {
+      provide: HTTP_INTERCEPTORS,
+      useClass: TokenInterceptorService,
+      multi: true,
+    }
+  ],
```

Comment:
As mentioned earlier, this provider should be vomed inside `CoreModule` instead.